rollrus.go

```diff
@@ -145,22 +145,20 @@ func setupLogging(token, env string, levels []logrus.Level) {
 	}
 }
 
-// ReportPanic attempts to report the panic to rollbar using the provided
+// ReportPanic attempts to report the panic to Rollbar using the provided
 // client and then re-panic. If it can't report the panic it will print an
 // error to stderr.
 func (r *Hook) ReportPanic() {
 	if p := recover(); p != nil {
-		if _, err := r.Client.Critical(fmt.Errorf("panic: %q", p), nil); err != nil {
-			fmt.Fprintf(os.Stderr, "reporting_panic=false err=%q\n", err)
-		}
+		r.Client.ErrorWithLevel(rollbar.CRIT, fmt.Errorf("panic: %q", p))
```

Comment:
I think there should be r.Client.Wait before the final panic() to make sure that the error actually reaches Rollbar.